Portfolio2.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+	<link rel="stylesheet" href="portfolio2.css" type="text/css">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link <link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Ubuntu:wght@300;400;500;700&display=swap" rel="stylesheet">
+	<script src="https://kit.fontawesome.com/2341878589.js" crossorigin="anonymous"></script>
+	
+    <title>Richard's Portfolio'</title>
+  </head>
+
+<body>
+
+  <nav class= "container">
+	  	<img src="https://images.pexels.com/photos/5044497/pexels-photo-5044497.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260")>
+	 <div class="links">
+		  <a href="#about">About Me</a>
+		  <a href="#portfolio">Richard's Portfolio</a>
+		  <a href="#contact">Contact Me</a>
+	  </div>
+  </nav>
```

Comment:
inconsistent indentation, switches between spaces and tab. pick one and stick to it :)